gitlab-pipeline/stage/build.yml

```diff
@@ -293,6 +293,12 @@ build:generate-delta-worker:docker:
     - mkdir -p "${WORKSPACE}"
     - cd "${WORKSPACE}"
     - tar -xf /tmp/workspace.tar.gz
+
+    - if ! ${WORKSPACE}/integration/extra/release_tool.py -l | grep -qF generate-delta-worker; then
```

Comment:
and where are my double quotes around the var? :>

Reply:
Ah crap, good eyes! Fixing! Looks like it worked though. The failure is for a different reason.